src/scope.mjs

```diff
@@ -91,15 +94,15 @@ export class Scope {
 			}
 			// Horizontal grid
 			const dbRange = this.maxDecibels - this.minDecibels;
-			for(let i = 10; i < dbRange; i += 10) {
-				const y = i * height / dbRange;
+			for(let i = 10; i <= dbRange; i += 10) {
+				const y = (i - 5) * height / dbRange;
```

Comment:
Did you decide to shift the grid and change the labels? The range was -120 to -20 and was tied to the analyzer settings. Now it's -105 to -5. Why -5 at the top?